msg.c

```diff
@@ -60,27 +119,43 @@ static void color_log(FILE* f, const char* color, const char* fmt, va_list vl)
     }
 }
 
+static void logger(enum LOG_LEVEL level, const char* fmt, va_list vl)
+{
+    if (use_syslog) {
+        return vsyslog(log_level_syslog(level), fmt, vl);
+    }
+
+    color_log(log_level_stream(level), log_level_color(level), fmt, vl);
+}
+
+void earlyoom_syslog_init(void)
+{
+    if (!use_syslog) {
```

Comment:
Do we need this `if`?

Reply:
just double-init check, which is does not leads to any problems, but anyway.